lib/taskana-core/src/main/java/pro/taskana/workbasket/internal/jobs/WorkbasketCleanupJob.java

```diff
@@ -76,31 +76,27 @@ public static void initializeSchedule(TaskanaEngine taskanaEngine) {
   }
 
   private List<String> getWorkbasketsMarkedForDeletion() {
-    List<String> workbasketList =
-        taskanaEngineImpl
-            .getWorkbasketService()
-            .createWorkbasketQuery()
-            .markedForDeletion(true)
-            .listValues(WorkbasketQueryColumnName.ID, BaseQuery.SortDirection.ASCENDING);
 
-    return workbasketList;
+    return taskanaEngineImpl
+        .getWorkbasketService()
+        .createWorkbasketQuery()
+        .markedForDeletion(true)
+        .listValues(WorkbasketQueryColumnName.ID, BaseQuery.SortDirection.ASCENDING);
   }
 
   private int deleteWorkbasketsTransactionally(List<String> workbasketsToBeDeleted) {
     int deletedWorkbasketsCount = 0;
     if (txProvider != null) {
-      int count =
-          (Integer)
-              txProvider.executeInTransaction(
-                  () -> {
-                    try {
-                      return deleteWorkbaskets(workbasketsToBeDeleted);
-                    } catch (Exception e) {
-                      LOGGER.warn("Could not delete workbaskets.", e);
-                      return 0;
-                    }
-                  });
-      return count;
+      return (int) (Integer)
```

Comment:
Why are you additionally casting from Integer to int? Autounboxing should do this automatically